docs/src/content/components/steps.mdx

```diff
@@ -0,0 +1,265 @@
+---
+name: Steps
+route: /components/steps
+---
+
+import Steps from '@pluralsight/ps-design-system-steps'
+
+# Steps
+
+<Intro>Use the Steps component to display sequential navigation. A step will display an incremental counter and may have a title and description.</Intro>
+
+<TableOfContents {...props} />
+
+## Examples
+
+### Orientations
+
+The `Steps` component can be oriented vertically or horizontally.
+
+```typescript
+import React from 'react'
+import Steps from '@pluralsight/ps-design-system-steps'
+
+function Example() {
+  return (
+    <Steps orientation="vertical">
+      <Steps.Step status={Steps.statuses.completed}>
+        Finished
+      </Steps.Step>
+
+      <Steps.Step
+        description="An optional description to provide more detail about this step."
+        status={Steps.statuses.current}
+      >
+        In Progress
+      </Steps.Step>
+
+      <Steps.Step status={Steps.statuses.incomplete}>
+        Waiting
+      </Steps.Step>
+    </Steps>
+  )
+}
+
+export default Example
+```
+
+```typescript
+import React from 'react'
+import Steps from '@pluralsight/ps-design-system-steps'
+
+function Example() {
+  return (
+    <Steps orientation="horizontal">
+      <Steps.Step status={Steps.statuses.completed}>
+        Finished
+      </Steps.Step>
+
+      <Steps.Step
+        description="An options description to provide more detail about this step."
+        status={Steps.statuses.current}
+      >
+        In Progress
+      </Steps.Step>
+
+      <Steps.Step status={Steps.statuses.incomplete}>
+        Waiting
+      </Steps.Step>
+    </Steps>
+  )
+}
+
+export default Example
+```
+
+### Sizes
+
+The `size` prop will adjust the font and marker dimensions.
+
+```typescript
+import React from 'react'
+import Steps from '@pluralsight/ps-design-system-steps'
+
+function Example() {
+  return (
+    <>
+      <Steps orientation="horizontal" size="medium">
+        <Steps.Step status={Steps.statuses.completed}>Medium</Steps.Step>
+      </Steps>
+
+      <Steps orientation="horizontal" size="large">
+        <Steps.Step status={Steps.statuses.completed}>Large</Steps.Step>
+      </Steps>
+    </>
+  )
+}
+
+export default Example
+```
+
+### Interactive
```

Comment:
I like that this is a pattern.  Feels good to put this state management in the app.  Esp since these are probably longer-lived states tracked over the wizard-like experiences.